src/tests/pages/place/[placeId]/status.test.js

```diff
@@ -0,0 +1,76 @@
+import { place as mockedPlace, place } from '../../../data/place';
```

Comment:
```suggestion
import { place as mockedPlace } from '../../../data/place';
```

Reply:
I'm fixing this in a separate commit since it's not mockedPlace we're using everywhere